BusBoard.ConsoleApp/Program.cs

```diff
@@ -3,13 +3,44 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RestSharp;
+using RestSharp.Authenticators;
 
 namespace BusBoard.ConsoleApp
 {
   class Program
   {
     static void Main(string[] args)
     {
+      Console.WriteLine("Please enter your desired stop point ID:");
+      var stopCode = Console.ReadLine();
+      Console.WriteLine(" ");
+      var response = GetListOfArrivalPredictionsForStopPoint(stopCode);
+      var buses = GetNext5Buses(response);
+      Console.WriteLine($"Next 5 Buses at stop {stopCode}");
+      foreach (var bus in buses)
+      {
+        Console.WriteLine($"VehicleID: {bus.VehicleId}, ETA: {bus.ExpectedArrival.AddHours(1).ToString("HH:mm:ss tt")}");
+      }
+      Console.WriteLine(" ");
+    }
+    
+
+    private static IRestResponse GetListOfArrivalPredictionsForStopPoint(string stopCode)
```

Comment:
I also think you should pull out the API logic in to a separate class. Thinking in terms of separation of concerns it would be nice to have a `TflApi` class that handles all the details